CHANGELOG.md

```diff
@@ -21,9 +47,10 @@ and this repository adheres to [Semantic Versioning](https://semver.org/spec/v2.
 ### Enhancements
 
 - We have added instrumentation for [HikariCP](https://github.com/brettwooldridge/HikariCP)
-  and [Tomcat JDBC](https://tomcat.apache.org/tomcat-8.5-doc/jdbc-pool.html) connection pools. The agent now
-  collects and exports metrics for both JDBC connection pools.
-- You can now set the service name using the `OTEL_SERVICE_NAME` environment variable and the `otel.service.name` system property. This removes the need of using `OTEL_RESOURCE_ATTRIBUTES` to set the service name.
+  and [Tomcat JDBC](https://tomcat.apache.org/tomcat-8.5-doc/jdbc-pool.html) connection pools. The agent now collects
+  and exports metrics for both JDBC connection pools.
+- You can now set the service name using the `OTEL_SERVICE_NAME` environment variable and the `otel.service.name` system
```

Comment:
Would be nice to link to the spec here (https://github.com/open-telemetry/opentelemetry-specification/pull/1677 or https://github.com/open-telemetry/opentelemetry-specification/blob/dfd4039a4fc178da89dc2e9eaefd79f651604fa3/CHANGELOG.md#sdk-configuration-3 or https://github.com/open-telemetry/opentelemetry-specification/blob/c5f1a70340059953244fcc9aabb371347e0989fc/specification/sdk-environment-variables.md#general-sdk-configuration not sure which is best)